download_and_create_reference_datasets/v02/hail_scripts/write_combined_reference_data_ht.py

```diff
@@ -217,6 +229,28 @@
 CONFIG['dbnsfp_mito']['38']['filter'] = lambda ht: ht.locus.contig == 'chrM'
 
 
+def annotate_ref_regions(ht, region_dataset, reference_genome):
+    """
+    Annotate combined reference data with region based reference data.
+    :param ht: Combined reference hail Table.
+    :param region_dataset: Imported region/bed file with 3 or 4 columns
+    :param reference_genome: Reference genome, e.g. GRCh38
+    :return: hail Table annotated with references target annotation.
+    """
+    region_ht = hl.read_table(CONFIG[region_dataset][reference_genome]["path"])
+    region_select = {
+        region_dataset: hl.struct(
+            **{
+                CONFIG[region_dataset][reference_genome]["select"]["target"]: region_ht[
+                    ht.locus
+                ].target
```

Comment:
I'm curious why we split these into multiple lines.

Reply:
Ah in the gnomAD repos we format using the tool 'black' which only allows lines to be a certain length. I think I formatted this file the same way. I can make this a single line but I do plan on formatting the entire repo using black at some point as part of a standardization process.